utils/__tests__/notification.vanilla.test.ts

```diff
@@ -0,0 +1,35 @@
+// tslint:disable-next-line: no-object-mutation
+process.env = {
+  ...process.env,
+  AZURE_NH_ENDPOINT:
+    "Endpoint=sb://127.0.0.1:30000;SharedAccessKeyName=DefaultFullSharedAccessSignature;SharedAccessKey=foobar",
+  AZURE_NH_HUB_NAME: "io-notification-hub-mock"
+};
```

Comment:
it's (almost) never a good idea to override node globals for testing. Instead, you can mock the config module.
https://jestjs.io/docs/manual-mocks

Reply:
Another piece of code I copied.  I'll refactor it..

Reply:
These values 
`FETCH_KEEPALIVE_ENABLED: "true",
FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT: "30000",
FETCH_KEEPALIVE_MAX_FREE_SOCKETS: "10",
FETCH_KEEPALIVE_MAX_SOCKETS: "40",
FETCH_KEEPALIVE_SOCKET_ACTIVE_TTL: "110000",
FETCH_KEEPALIVE_TIMEOUT: "60000"`

are not included into the config file (and within IConfig type, not sure what should I do here

Reply:
They are red by the http client we use. cc @gunzip

Reply:
Sorry, I explained myself badly. I was aware that those values are used (I saw the terraform as well), I don't know how to treat them in this particular test